src/NUnitEngine/nunit.engine/Runners/ProcessRunner.cs

```diff
@@ -80,7 +80,7 @@ public override TestEngineResult Explore(TestFilter filter)
             }
             catch (Exception e)
             {
-                log.Error("Failed to run remote tests {0}", e.Message);
+                log.Error("Failed to run remote tests {0}", ExceptionHelper.BuildStackTrace(e));
```

Comment:
Slightly surprised to learn that `BuildStackTrace` includes the message as well.

Reply:
Me too, I'll rename